custom/conf/app.ini.sample

```diff
@@ -39,6 +39,12 @@ ACCESS_CONTROL_ALLOW_ORIGIN =
 USE_COMPAT_SSH_URI = false
 ; Close issues as long as a commit on any branch marks it as fixed
 DEFAULT_CLOSE_ISSUES_VIA_COMMITS_IN_ANY_BRANCH = false
+; Create Table of Contents based on headlines on wiki page (does not alter file)
```

Comment:
Please add these items to `markdown` section

Reply:
I've already added this to docs/content/doc/advanced/config-cheat-sheet.en-us.md
Do I need to mention this somewhere else?

Reply:
done